docs/serving/autoscaling/autoscaling-targets.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Targets"
+linkTitle: "Targets"
+weight: 04
+type: "docs"
+---
+
+Configuring a target provide the Autoscaler with a value that it tries to maintain for the configured metric for a revision.
+See the [metrics](./autoscaling-metrics.md) documentation for more information about configurable metric types.
+
+The `target` annotation, used to configure per-revision targets,  is _metric agnostic_. This means the target is simply an integer value, which can be applied for any metric type.
+
+## Configuring targets
+
+* **Global settings key:** `container-concurrency-target-default` for setting a concurrency target, and `requests-per-second-target-default` for setting a requests-per-second (RPS) target. For more information, see the documentation on [metrics](./autoscaling-metrics.md).
+* **Per-revision annotation key:** `autoscaling.knative.dev/target`
+* **Possible values:** An integer (metric agnostic).
+* **Default:** `"50"` for the `target` annotation, `"100"` for `container-concurrency-target-default`, and `"200"` for `requests-per-second-target-default`.
```

Comment:
I'm not sure I understand how the `target` and `-default` defaults work together?

@vagababov

Reply:
FWIW: There's no default for the target annotation. There are global default settings per metric as shown in the global settings key.